### DISCUSSION TITLE
Specfem and strain

### DISCUSSION THREAD

**Discussion by nlscelli** [SPECFEM/specfem3d#1684] [General]
Hi, I was wondering if there is any current plan/ongoing work to implement the direct measurement of strain in one of the packages.
Cheers,

Nicolas Celli (Postdoc-DIAS)

**Comment by danielpeter** [comment]
the 3D packages, SPECFEM3D_GLOBE  and SPECFEM3D_Cartesian, both have an option `SAVE_SEISMOGRAMS_STRAIN` in the `Par_file` to output strain at the receiver locations.
